cluster/up.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -ex
+
+source ./cluster/kubevirtci.sh
+kubevirtci::install
+
+$(kubevirtci::path)/cluster-up/up.sh
+
+if [[ "$KUBEVIRT_PROVIDER" =~ ^(okd|ocp)-.*$$ ]]; then \
+		while ! $(KUBECTL) get securitycontextconstraints; do sleep 1; done; \
+fi
+
+if [[ "$KUBEVIRT_PROVIDER" =~ k8s- ]]; then
+    echo 'Install NetworkManager on nodes'
+    for node in $(./cluster/kubectl.sh get nodes --no-headers | awk '{print $1}'); do
+        ./cluster/cli.sh ssh ${node} sudo -- yum install -y yum-plugin-copr
+        ./cluster/cli.sh ssh ${node} sudo -- yum copr enable -y networkmanager/NetworkManager-1.22-git
```

Comment:
rest of the projects (for example CNAO) still use 1.20

Reply:
Yes. What are you suggesting?

Reply:
just raising in order to think together if it is even a problem, or to take into consideration (well known i guess)

Reply:
knmstate is dependent on NM. We want to test version of NM that will be included on the platform we will release current master on. So we definitely want to test 1.22 here. For other projects, it does not matter that much, but eventually we will likely converge to 1.22.

Reply:
thanks, 
wondering - out of this PR, since those scripts are partly common between projects
wouldn't it be good to make them in a seperate repo, vendor them and have some VARS
for example for this case the NM versions that need to be distinct between the different projects ?

Reply:
They are already sharing the main common part - kubevirtici. I am hesitant to make the rest of the code shared. It depends heavily on the project. That's obvious from cluster-clean and cluster-sync. For cluster-up, we usually install additional components (e.g. NM or OVS). Having it duplicated is pretty convenient. If we would want to keep it on a common place, we need to think it through.

Reply:
cant press resolve, sgtm , thanks, added rocket instead resolve